__tests__/selectors.test.js

```diff
@@ -1,6 +1,79 @@
+import { getWaiter, getWaiterResponse, getWaiterError } from '../src'
+import { waiterModel } from '../src/waiter-reducer'
+import REDUX_MOUNT_POINT from './redux-mount-point'
 
 describe('Selectors', () => {
-  it('should be true', () => {
-    expect(true).toBe(true)
+  describe('getWaiter', () => {
+    it('Should return waiter model if namespace does not exist', () => {
+      const state = {
+        [REDUX_MOUNT_POINT]: {},
+      }
+      expect(getWaiter(state, 'mywaiter')).toEqual(waiterModel)
```

Comment:
I realize this test doesn't really protect against changes in the `waiterModel` since it's being pulled in from `waiter-reducer`, but it still verifies that the model is returned for namespaces that don't exist -- so I figured the test wasn't entirely without merit.

Reply:
makes sense to me